docker-compose.repo.yml

```diff
@@ -3,7 +3,7 @@ version: "3"
 services:
   desci_repo:
     container_name: "desci_repo"
-    entrypoint: /bin/sh './scripts/be-node-dev.sh'
```

Comment:
does debugging still work with this change?

Reply:
It never worked for desci_repo, I always had an error like "be-node-dev.sh not found". But the service worked regardless, so I cleaned it up since it was unused.

Reply:
Additionally, the only thing it actually did was to run `yarn dev`, so I simplified with changing to this in `docker-compose.repo.yml`:
```
command: yarn dev
```